system/atomic_image.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION='''
+---
+module: atomic_image
+short_description: Manage the container images on the atomic host platform
+description:
+    - Manage the container images on the atomic host platform
+    - Allows to execute the commands on the container images
+    - Commands ```install``` and ```run``` will download the container image if not present already
+version_added: "2.2"
+author: "Saravanan KR @krsacme"
+notes:
+    - Host should be an atomic platform (verified by existence of '/run/ostree-booted' file)
+requirements:
+  - atomic
+options:
+    name:
+        description:
+          - Name of the container image
+        required: True
+        default: null
+    upgrade:
+        description:
+          - Forcefully upgrade if the container, even if the container is already running
+        required: False
+        choices: ["yes", "no"]
+        default: no
+    state:
+        description:
+          - The state of the container image
+        required: False
+        choices: ["started", "stopped"]
```

Comment:
@krsacme thanks for the context.

@tima I think I would have to disagree with not wanting a module to do both deployment and service management, since those things are so intertwined at the container level currently, it would seem odd to me to have to specify that I want a particular version of a container image installed and then have to separately say that I want that specific version of that container image to also be running (since there could be multiple versions of a given container present).

@chouseknecht I'm curious to your take on the situation, since ansible-container faces similar issues.